Guided Investigation - MDATP Webshell Alerts.ipynb

```diff
@@ -0,0 +1,700 @@
+{
```

Comment:
<p>We have added a common install and setup cell for all our notebooks (check out any of the explorer notebooks) that you could use here if you like but this isn't a requirement. However the most common issues we have raised by users relate to import and set up so its advisable.</p><p> </p><p>One thing that would be nice regardless is to avoid msticpy installation if possible as it can take a long time on the free Azure Notebooks tier. You could do this simply with:</p><p>try:</p><p>import msticpy</p><p>execpt ModuleNotFoundError:</p><p>!pip install msticpy --upgrade --user</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Thanks Pete, I will take your advice and use the standard import template, thanks for pointing me in the right direction.</p>